### ISSUE TITLE
Migrate requirements.txt to environment.yaml

### ISSUE THREAD

**Issue by kaedonkers** [MetOffice/forest#446]
Using Conda's yaml setup for the packages allows more concise specification along with naming and channel specs

**Comment by kaedonkers** [comment]
For example:
```txt
bokeh
nodejs>=10.13
datashader
h5netcdf
iris<3
intake
intake-esm
pygrib
xarray
geojson
pyyaml
pandas
cftime
scipy
netCDF4
```
becomes
```yaml
name: forest
channels:
  - conda-forge
dependencies:
  - bokeh
  - nodejs>=10.13
  - datashader
  - h5netcdf
  - iris<3
  - intake
  - intake-esm
  - pygrib
  - xarray
  - geojson
  - pyyaml
  - pandas
  - cftime
  - scipy
  - netCDF4
```

**Comment by andrewgryan** [comment]
Closed by #450
